openbas-front/src/admin/components/common/injects/UpdateInjectDetails.js

```diff
@@ -355,7 +354,6 @@ const UpdateInjectDetails = ({
                       handleClose={handleClose}
                       tagsMap={tagsMap}
                       permissions={permissions}
-                      teamsFromExerciseOrScenario={teamsFromExerciseOrScenario}
                       articlesFromExerciseOrScenario={[]}
```

Comment:
Maybe it's possible to replace/remove  fetchTeams (lines 73) ?

Reply:
Remote it